client/hotrod-client/src/main/java/org/infinispan/client/hotrod/logging/Log.java

```diff
@@ -387,9 +387,9 @@ public interface Log extends BasicLogger {
    @Message(value = "Native IOUring transport not available, using NIO instead: %s", id = 4108)
    void ioUringNotAvailable(String cause);
 
-   @LogMessage(level = DEBUG)
+   @LogMessage(level = TRACE)
```

Comment:
DEBUG for this is okay now that it is only logged once and without a stack trace.